dts/arm/atmel/same70b.dtsi

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020 Stephanos Ioannidis <root@stephanos.io>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <atmel/same70.dtsi>
+
+&gmac {
+	interrupts = <39 0>, <66 0>, <67 0>, <71 0>, <72 0>, <73 0>;
+	interrupt-names = "gmac", "q1", "q2", "q3", "q4", "q5";
+	queue-num = <6>;
+};
```

Comment:
@galak could you double check if this syntax is OK or we should prefer
```
/ {
	soc {
		gmac: ethernet@40050088 {
			...
		};
	};
};
```

Reply:
They mean two different things: what you suggested above instantiates a new device named "gmac", while that code sets properties for the already-instantiated device named "gmac".

Reply:
Note that the device `gmac` was already instantiated by `same70.dtsi` at this point.

Reply:
I'm fine w/the syntax choice.  No real preference on how to do this in the soc dtsi files.  Leave it to the soc maintainer to decide what style to use.